mean_test.hhs

```diff
@@ -0,0 +1,21 @@
+/**
+ * @author Jianan Lin (林家南)
+ */
+
+*import math: mean
+
+let a = 1;
+let b = [5,34,3];
+let c = [[3,2,3],[3,4,2]];
+let d = new Mat([3,4,2]);
+let e = new Mat([[3,4],[4,3],[4,3]]);
+let f = [[[2,2],[3,3],[4,4]],[[2,2],[3,3],[4,4]],[[4,4],[3,3],[2,2]]];
+let g = [[1, 2], [3], [4, 5, 6]];
+
+print(mean(1, 2));
```

Comment:
The way we've been doing unit test cases are like this:

You already declared some variables to work with. Now throw errors if the answers don't match the expected answers:

Go to wolfram or whatever, and find the mean of each one. for a = 1, we expect mean=1. So instead of 
`print(mean(a))`
do,
```js
if ( !(mean(a) === 1) ) {
    throw new Error('Unit test case failed for a = 1');
}
```